ci/build.sh

```diff
@@ -18,6 +18,31 @@ function get_branch_to_checkout() {
     || echo "${TRAVIS_BRANCH}"
 }
 
+function bundle_fluentd_plugin() {
+  local plugin_name="${1}"
+  local version="${2}"
+  # Strip everything after "-" (longest match) to avoid gem prerelease behavior
+  local gem_version="${version%%-*}"
+
+  cd "${plugin_name}" || exit 1
+
+  if [[ "${version}" == "" ]] ; then
```

Comment:
`== ""` vs `-z` ? Not sure which one we prefer (a couple of other scripts use `-z`)

Reply:
Hm `-z` is not a look-and-know whereas `== ""` is 🤔

Reply:
I go with the `-z` as it was designed for that usage 🤷

Reply:
look-and-know doesn't apply to the bash anyway 🤔